### ISSUE TITLE
to_proto() has unexpected side-effects

### ISSUE THREAD

**Issue by mrtnlrsn** [dfpc-coe/node-CoT#55]
Calling `to_proto()` modifies the object. That is unexpected and means the object  cannot be used after `to_proto()`

E.g. calling `to_proto()` twice, to send the message on 2 conncetions, results in the 2nd send to not include callsign. The last assert in the following test fails due to that.

```ts
        const cot = new CoT({
            event: {
                _attributes: {
                    version: '2.0',
                    uid: 'ebbf42a7-ea71-43a1-baf6-e259c3d115bf',
                    type: 'u-rb-a',
                    how: 'h-e',
                    time: '2024-08-30T22:28:02Z',
                    start: '2024-08-30T22:28:02Z',
                    stale: '2024-08-31T22:28:02Z',
                    access: 'Undefined',
                },
                point: {
                    _attributes: {
                        lat: '39.0981196',
                        lon: '-108.7395013',
                        hae: '0.0',
                        ce: '9999999.0',
                        le: '9999999.0',
                    },
                },
                detail: {
                    contact: {
                        _attributes: {
                            callsign: 'sign',
                        },
                    },
                },
            },
        })

        const cot2 = CoT.from_proto(cot.to_proto())
        assert.deepEqual(cot2.raw.event.detail?.contact?._attributes.callsign, 'sign')
        const cot3 = CoT.from_proto(cot.to_proto())
        assert.deepEqual(cot3.raw.event.detail?.contact?._attributes.callsign, 'sign')
```

**Comment by ingalls** [comment]
I'll take a look at this first thing next week

**Comment by ingalls** [comment]
Released - v12.16.2 

Thanks as always for the report!
